modules/core/image/farm_image.install

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall function for the farm_image module.
+ */
+
+declare(strict_types=1);
+
+use Drupal\image\Entity\ImageStyle;
+
+/**
+ * Implements hook_install().
+ */
+function farm_image_install() {
+
+  // Add auto-orient effect to Drupal core image styles.
+  $styles = [
+    'large',
+    'medium',
+    'thumbnail',
+    'wide',
+  ];
+  foreach ($styles as $name) {
+    $style = ImageStyle::load($name);
+    $style->addImageEffect([
```

Comment:
Is there any argument for being more defensive here - i.e. if one style doesn't exist or fails to load, should it fail entirely or proceed to the others?

Reply:
Good call - fixup commit: https://github.com/farmOS/farmOS/pull/941/commits/1ec09787d03eda62b5ab60ca2976c286d6b96187